code/index.html

```diff
@@ -3,28 +3,252 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>📚 Bella's Book Club</title>
```

Comment:
You can look into having an icon or favicon that will exchange the default icon (in my case 🌎) instead of having an emoji in the title 😎 
<img width="119" alt="Skärmavbild 2021-02-09 kl  10 01 45" src="https://user-images.githubusercontent.com/51529201/107340247-d2a00e00-6abd-11eb-8ce7-391bfb974f1e.png">